src/services/models/MediaType.ts

```diff
@@ -31,6 +34,9 @@ export class MediaTypeModel {
     this.schema = info.schema && new SchemaModel(parser, info.schema, '', options);
     this.onlyRequiredInSamples = options.onlyRequiredInSamples;
     this.generatedPayloadSamplesMaxDepth = options.generatedPayloadSamplesMaxDepth;
+    const isCodeGenerationSupported = options.codeSamplesLanguages.some(lang =>
```

Comment:
Remember we had a linting rule for this in the app. Do you have any plans to keep such rules consistent for this repo?

```suggestion
    const isCodeGenerationSupported = options.codeSamplesLanguages.some((lang) =>
```

Reply:
No for this repo we are following Redoc's linting rules and conventions set out by prettier, with the intention of creating a PR against their base repo.